tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -264,6 +277,19 @@
 export PKG_CONFIG_PATH
 export PYTHON_HOME
 export _BPXK_AUTOCVT""",
+                  test_uss_block_insert_with_doublequotes="""//BPXSLEEP JOB MSGCLASS=A,MSGLEVEL=(1,1),NOTIFY=&SYSUID,REGION=0M
```

Comment:
This is not really a show stopper but if you wanted to generate valid JCL it would look like:
```
//BPXSLEEP JOB MSGCLASS=A,MSGLEVEL=(1,1),NOTIFY=&SYSUID,REGION=0M
//USSCMD EXEC PGM=BPXBATCH
//STDERR  DD SYSOUT=*
//STDOUT  DD SYSOUT=*
//STDPARM DD *
SH ls -la /;
sleep 30;
# BEGIN ANSIBLE MANAGED BLOCK
cat "//OMVSADMI.CAT"
cat "//OMVSADM.COPYMEM.TESTS" > test.txt
# END ANSIBLE MANAGED BLOCK
/*
//
```

Reply:
Going to approve but just putting this there for your own understanding, between the `//STDPARM DD *` and the `/*`, where you see `SH` this is just pure shell , so comments are now `#` where JCL comments are `//*`